scripts/sql/user_hssc_etl_create.sql

```diff
@@ -0,0 +1,25 @@
+/*
+ * After creating HSSC_ETL user, log in and create the private MPI DB Link:
+ 
+ CREATE DATABASE LINK "MPI"
```

Comment:
create database link MPI
  connect to <schema> identified by <password>
  using '(DESCRIPTION = 
    (ADDRESS = (PROTOCOL = TCP) (HOST = hssc-cdw-mpidb-d.clemson.edu)
    (PORT = 1521))
    (CONNECT_DATA = (SID = dtprd2))
    )';